InternxtDesktop/Views/Widget/WidgetFolderList.swift

```diff
@@ -67,7 +73,7 @@ struct WidgetFolderList: View {
     }
 
     private func getRowBackgroundColor(for index: Int) -> Color {
-        if selectedIndex == index {
+        if selectedId == foldernames[index].id {
```

Comment:
I don't understand this logic, if you have the ID of the row, just compare the given ID with the selectedId and that's it right? Am I missing something?

Reply:
i don't have the id of the row. i have the index, this is for rendering odd rows with some specific color and even rows with other color as it is on figma. 

this if statement is to check if you selected that specific id to delete it from the foldernames array or to open it on finder